benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1824,13 +1826,12 @@ void GraphicsBenchmarkApp::RecordCommandBufferFullscreenQuad(PerFrame& frame, si
             float3 colorValues = pFullscreenQuadsColor->GetValue();
             colorValues *= intensity;
             mQuadPushConstant.ColorValue = colorValues;
-            frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[0], GetPushConstCount(mQuadPushConstant.ColorValue), &mQuadPushConstant.ColorValue, offsetof(QuadPushConstant, ColorValue) / sizeof(uint32_t));
+            frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[int(pFullscreenQuadsType->GetValue())], GetPushConstCount(mQuadPushConstant.ColorValue), &mQuadPushConstant.ColorValue, offsetof(QuadPushConstant, ColorValue) / sizeof(uint32_t));
```

Comment:
nit: you can probably declare the enum with a type of uint32_t so that you don't need the cast:

enum class FullscreenQuadsType: uint32_t
{
FULLSCREEN_QUADS_TYPE_NOISE,
FULLSCREEN_QUADS_TYPE_SOLID_COLOR,
FULLSCREEN_QUADS_TYPE_TEXTURE
};

Reply:
I have just tried and it does not seem to work.
I did a quick Google serach, and some pages says that even if you specify the undelying type, you need cast to use as index.

Reply:
ah, thanks for trying this, good to know, and sorry for the wrong information. 
also better to use `static_cast` for such casting those as suggested by the cpp styple guide: go/cstyle#Casting